docs/csharp/toc.yml

```diff
@@ -93,6 +95,7 @@
 - name: What's new in C#
   items:
   - name: C# 8.0
+    displayName: what's new
```

Comment:
Why lower-case?

```suggestion
    displayName: What's new
```

Reply:
Despite the name, this text isn't displayed anywhere, so casing is irrelevant.